.github/workflows/build_package.yml

```diff
@@ -150,13 +150,13 @@ jobs:
           # MacOS packages.
           # TODO(scotttodd): build on larger runner when available (self-hosted or GitHub)
           # - runs-on:
-          #     - ${{ github.repository == 'iree-org/iree' && 'self-hosted' || 'macos-14' }}
+          #     - ${{ github.repository == 'iree-org/iree' && 'self-hosted' || 'macos-15' }}
           #     - os-family=macOS
-          - runs-on: macos-14
+          - runs-on: macos-15
             build-family: macos
             build-package: py-compiler-pkg
             experimental: true
-          - runs-on: macos-14
+          - runs-on: macos-15
```

Comment:
These may be riskier around release time (right now), if not tested ahead of merging. Let's wait a few days (after whatever release cut we pick) to merge but otherwise LGTM. Could also split the changes in this file into another PR.

Reply:
Fair enough. I removed those changes from this PR.